idaes/gas_solid_contactors/properties/methane_iron_OC_reduction/gas_phase_thermo.py

```diff
@@ -532,20 +535,21 @@ def _visc_d(self):
 
         def visc_d_comp(i):
             return self._params.visc_d_param[i, 1] * \
-                    (self.temperature**self._params.visc_d_param[i, 2]) \
-                    / ((1 + (self._params.visc_d_param[i, 3]/self.temperature))
+                    ((self.temperature/pyunits.K) **
+                     self._params.visc_d_param[i, 2]) \
+                    / ((1 + (self._params.visc_d_param[i, 3] /
+                             (self.temperature/pyunits.K)))
```

Comment:
Yes, the 1/Kelvin units are to address unit inconsistency issues.

Reply:
I think you need to be a bit more careful here. The first reference to temperature needs to have the units removed as it is raised to a variable, non-integer power. However, in the later terms the parameters should have units which cancel with T - e.g. parameter 3 should have units of `1/K` which will cancel with those of `temperature` as required. Similar for parameter 4 (units should be 1/K**2).

Note that this required you to create separate `Vars` for each parameter (rather than a single indexed parameter) as all elements of an indexed `Var` must have the same units. However, this is what we are doing everywhere else in IDAES, and is probably the better way to do this.

Reply:
I'm reviewing property packages with Parameters for each coefficient, indexed by component, for proper syntax. Would you recommend modifying the gas/solid Cp and gas thermo_cond parameters in this way as well?

Reply:
It needs to happen at some point. However, if the PR starts getting too big you might want to try breaking it up into stages.

Reply:
This PR is focusing only on the FixedBed0D model and related updates  to the methane iron OC properties. There will be other PRs for the MovingBed and BubblingFluidizedBed models, and I could leave the Param updates until then to simplify this PR.

Reply:
I think that's the approach to take. @bpaul4 the gas proppack file we discussed at our last meeting has the cp param, and enthalpy and cp_mol variables setup in this manner. Please take a look when you get the opportunity.